docs/index.md

```diff
@@ -86,6 +88,16 @@ Example applications are also provided in R. For example,
 
     ./bin/spark-submit examples/src/main/r/dataframe.R
 
+## Running Spark Client Applications Anywhere with Spark Connect
+
+Spark Connect is a new client-server architecture introduced in Spark 3.4 that decouples Spark
+client applications and allows remote connectivity to Spark clusters. The separation between
+client and server allows Spark and its open ecosystem to be leveraged from anywhere, embedded
+in any application. In Spark 3.4, Spark Connect provides DataFrame API coverage for PySpark.
+Spark Connect clients for other languages are planned for the future.
+
+To learn more about Spark Connect and how to use it, visit [Spark Connect Overview](spark-connect-overview.html).
```

Comment:
qq: It seems that the only way to access the Spark Connect Overview is through the link provided at this point.
Do you think that's enough, or maybe should we also place the link in a more prominent location to expose it more actively to the users?